### PR TITLE
FW-5368 and FW-5380 Related video links support

### PR DESCRIPTION
**Pull request by NolanVH**
### Description of Changes
This PR adds related video links support to the dashboard media browser/forms ([FW-5368](https://firstvoices.atlassian.net/browse/FW-5368)) and the places where uploaded videos can be viewed ([FW-5380](https://firstvoices.atlassian.net/browse/FW-5380)).

Related video links can be added to dictionary entries, songs, stories, and alphabet characters.

When a video link is added to the field, before saving the URL to the backend, a thumbnail is generated by sending a request to the YouTube or Vimeo APIs. If the field has already been saved in the backend then the thumbnail URL provided by the backend is used.

### Checklist

- ~README / documentation has been updated if needed~
- [x] PR title / commit messages contain Jira ticket number if applicable
- ~Tests have been updated / created if applicable~

### Reviewers Should Do The Following:

- [x] Review the changes here
- [x] Pull the branch and test locally

### Additional Notes

N/A

[FW-5368]: https://firstvoices.atlassian.net/browse/FW-5368?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[FW-5380]: https://firstvoices.atlassian.net/browse/FW-5380?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by sahil97 — APPROVED** [review]
Other than the bug discussed in chat, LGTM !
Nice work and thank you for re-iterating on this addition.

File: src/common/icons/Remove.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import PropTypes from 'prop-types'
```
Comment (sarahfirstvoices) [inline src/common/icons/Remove.js:2]: Is this different from the existing Trash.js? Do we need a separate one?

File: src/services/api/index.js [hunk 2]
```diff
@@ -18,6 +18,7 @@ import songs from 'services/api/songs'
 import stats from 'services/api/stats'
 import stories from 'services/api/stories'
 import storyPages from 'services/api/storyPages'
+import vimeoThumbnail from 'services/api/videoLinkThumbnails'
```
Comment (sarahfirstvoices) [inline src/services/api/index.js:21]: typo? vimeo vs video

**Review by bronson-c — APPROVED** [review]
Tested locally with all entry types, LGTM!

File: src/common/icons/Remove.js (hunk 1 above)
Comment (NolanVH) [reply src/common/icons/Remove.js:2]: It is slightly different. Trash.js is a trashcan and this is a circle with an X in the middle which matches the shape of the circle with a checkmark in the middle when a video is selected. Either could be used though.

File: src/services/api/index.js (hunk 2 above)
Comment (NolanVH) [reply src/services/api/index.js:21]: That specific API is to get the thumbnail from Vimeo videos only. The `videoLinkThumbnails` file could be used to add additional requests for other services in the future if needed.

File: src/common/icons/Remove.js (hunk 1 above)
Comment (gmcauliffe) [reply src/common/icons/Remove.js:2]: `TimesCircle` or `TimesCircleSolid` might be what you're after.

Comment (NolanVH) [reply src/common/icons/Remove.js]: Thank you both. I have removed this new icon and set the modal to use `TimesCircleSolid`.

**Pull request merged**